s-token/src/lib.rs

```diff
@@ -85,52 +88,23 @@ impl STokenTrait for SToken {
     /// - from - The address of the token owner.
     /// - spender - The address of the spender.
     /// - amount - The amount of tokens to increase the allowance by.
+    /// - expiration_ledger - The time when allowance will be expired.
     ///
     /// # Panics
     ///
     /// Panics if the caller is not authorized.
     /// Panics if the amount is negative.
     /// Panics if the updated allowance exceeds the maximum value of i128.
     ///
-    fn increase_allowance(e: Env, from: Address, spender: Address, amount: i128) {
+    fn approve(e: Env, from: Address, spender: Address, amount: i128, expiration_ledger: u32) {
```

Comment:
Do I understand correctly that the `approve` is now responsible for allowance?
I thought that the `approve` usually have different meaning - to approve some other party to transfer tokens.